upgrade_through_versions_test.py

```diff
@@ -188,7 +54,7 @@ def data_writer(tester, to_verify_queue, verification_done_queue, rewrite_probab
     Intended to be run using multiprocessing.
     """
     # 'tester' is a cloned object so we shouldn't be inappropriately sharing anything with another process
-    session = tester.patient_cql_connection(tester.node1, keyspace="upgrade", protocol_version=PROTOCOL_VERSION)
+    session = tester.patient_cql_connection(tester.node1, keyspace="upgrade", protocol_version=tester.protocol_version)
```

Comment:
Will have to dig back through this, but I think it shook out this way to deal with inappropriate driver object sharing between processes. Definitely a worthy place to improve, I'll see what I can do there.